### ISSUE TITLE
AlexNet前向传播速度慢

### ISSUE THREAD

**Issue by 133dzs** [PaddlePaddle/Paddle#39196]
1）PaddlePaddle版本：2.2.0
    2）CPU：R5-4600h
    3）系统环境：win10，python3.8
使用paddle写的alexnet前向传播速度很慢
网络结构：带batchnorm的5层alexnet
输入：[3,3,255,255]
测试每一层的运行时间，paddle版的时间是pytorch版的10倍左右。
paddle版时间：[0.173, 0.615, 0.743, 0.901, 0.951]
pytorch版时间：[0.026, 0.068, 0.082, 0.093, 0.096]
附代码：
self.conv1 = nn.Sequential(
    nn.Conv2D(3, 96, 11, 2),  
    nn.BatchNorm2D(96),
    nn.ReLU(),
    nn.MaxPool2D(3, 2))  
self.conv2 = nn.Sequential(
    nn.Conv2D(96, 256, 5, 1), 
    nn.BatchNorm2D(256),
    nn.ReLU(),
    nn.MaxPool2D(3, 2))  
self.conv3 = nn.Sequential(
    nn.Conv2D(256, 384, 3, 1),  
    nn.BatchNorm2D(384),
    nn.ReLU())
self.conv4 = nn.Sequential(
    nn.Conv2D(384, 384, 3, 1),  
    nn.BatchNorm2D(384),
    nn.ReLU())
self.conv5 = nn.Sequential(
    nn.Conv2D(384, 128, 3, 1))

**Comment by ronny1996** [comment]
hi，请问是使用CPU跑的吗？

**Comment by 133dzs** [comment]
> hi，请问是使用CPU跑的吗？

对，都是在笔记本的CPU上的测试结果，在服务器上的GPU环境速度没问题

**Comment by ronny1996** [comment]
你好，我在Intel(R) Xeon(R) Gold 6148 CPU上测试10次平均，paddle和torch没有10倍的差距，是否是数据io造成的？

```
# paddle
[0.097878   0.16446443 0.04221139 0.05142233 0.01911643]

# torch
[0.11108131 0.15792067 0.03854568 0.04771433 0.01271365]
```

**Comment by 133dzs** [comment]
> 你好，我在Intel(R) Xeon(R) Gold 6148 CPU上测试10次平均，paddle和torch没有10倍的差距，是否是数据io造成的？
> 
> ```
> # paddle
> [0.097878   0.16446443 0.04221139 0.05142233 0.01911643]
> 
> # torch
> [0.11108131 0.15792067 0.03854568 0.04771433 0.01271365]
> ```

非常感谢帮忙测试，我换了一台电脑测试，发现速度差距变小了一些，之后会再调试。
